From ce7b1bbc761ba451b120fc6498ebc5bf2b773fc4 Mon Sep 17 00:00:00 2001
From: Tim Parenti <tim@timtimeonline.com>
Date: Sat, 5 Dec 2020 23:38:05 -0500
Subject: [PATCH] Seychelles switched from LMT to +04 on 1907-01-01

(Thanks to P Chan.)
* NEWS: Mention this.
* africa (Indian/Mahe): Transition from LMT to standard time was
1907-01-01, not 1906-06.
---
 NEWS   |  1 +
 africa | 15 ++++++++++++++-
 2 files changed, 15 insertions(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index 0937d8a..dd131a9 100644
--- a/NEWS
+++ b/NEWS
@@ -18,6 +18,7 @@ Unreleased, experimental changes
       - Ghana: several 1915 through 1956 transitions
       - Israel and Palestine: several 1940 through 1985 transitions
       - Kenya and adjacent: several 1908 through 1960 transitions
+      - Seychelles: the introduction of standard time in 1907, not 1906
       - Vanuatu: DST in 1973-1974, and a corrected 1984 transition
     (Thanks to P Chan.)
 
diff --git a/africa b/africa
index 27c987a..e0200a9 100644
--- a/africa
+++ b/africa
@@ -1392,8 +1392,21 @@ Zone	Africa/Sao_Tome	 0:26:56 -	LMT	1884
 # See Africa/Abidjan.
 
 # Seychelles
+
+# From P Chan (2020-11-27):
+# Standard Time was adopted on 1907-01-01.
+#
+# Standard Time Ordinance (Chapter 237)
+# The Laws of Seychelles in Force on the 31st December, 1971, Vol. 6, p 571
+# https://books.google.com/books?id=efE-AQAAIAAJ&pg=PA571
+#
+# From Tim Parenti (2020-12-05):
+# A footnote on https://books.google.com/books?id=DYdDAQAAMAAJ&pg=PA1689
+# confirms that Ordinance No. 9 of 1906 "was brought into force on the 1st
+# January, 1907."
+
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
-Zone	Indian/Mahe	3:41:48 -	LMT	1906 Jun # Victoria
+Zone	Indian/Mahe	3:41:48 -	LMT	1907 Jan  1 # Victoria
 			4:00	-	+04
 # From Paul Eggert (2001-05-30):
 # Aldabra, Farquhar, and Desroches, originally dependencies of the
-- 
1.8.3.1

